src/ssh/azext_ssh/bastion_utils.py

```diff
@@ -0,0 +1,315 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+# pylint: disable=protected-access
+
+import platform
+import subprocess
+import tempfile
+import threading
+import time
+import json
+import uuid
+
+import requests
+from enum import Enum
+from azure.cli.core import azclierror 
+
+from azure.cli.core.commands.client_factory import get_subscription_id
+from knack.log import get_logger
+from azure.cli.core.aaz import *
+from knack.prompting import prompt_y_n
+
+
+from .aaz.latest.network.bastion import Create 
+from .vendored_sdks.resourcegraph._resource_graph_client import ResourceGraphClient
+from . import resource_graph_utils
+from . import ssh_utils
+
+logger = get_logger(__name__)
+
+
+class BastionSku(Enum):
+
+    Basic = "Basic"
+    Standard = "Standard"
+    Developer = "Developer"
+    QuickConnect = "QuickConnect"
+
+# ============================= Put Call to create a bastion Developer Sku============================= #
+def create_bastion(cmd, op_info, vnet_id):
+    try:
+        bastion_creator = Create(cli_ctx=cmd.cli_ctx)
+        bastion_args = {
+            "location": op_info.location,
+            "name": f"{op_info.resource_group_name}-vnet-bastion",
+            "resource_group": op_info.resource_group_name,
+            "sku": "Developer",
+            "virtual_network": {
+                "id": vnet_id
+            },
+            "ip_configurations": [],
+            "scale_units": 2,
+            "tags": {}
+        }
+        result = bastion_creator(command_args=bastion_args)
+         
+        op_info.bastion_name = f"{op_info.resource_group_name}-vnet-bastion"
```

Comment:
This doesn't work if a customer has two vnets in one resource group. vnet1 would have RG1-vnet-bastion and on creating a bastion in vnet2, the name would be RG1-vnet-bastion, resulting in a conflict. 
Instead, use "{vnet_name}-bastion.